src/main/java/com/openshift/internal/restclient/model/image/ImageStreamImport.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.openshift.internal.restclient.model.image;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Optional;
+
+import org.jboss.dmr.ModelNode;
+
+import com.openshift.internal.restclient.model.KubernetesResource;
+import com.openshift.internal.restclient.model.Status;
+import com.openshift.restclient.IClient;
+import com.openshift.restclient.images.DockerImageURI;
+import com.openshift.restclient.model.IStatus;
+import com.openshift.restclient.model.image.IImageStreamImport;
+
+/**
+ * 
+ * @author jeff.cantrill
+ *
+ */
+public class ImageStreamImport extends KubernetesResource implements IImageStreamImport {
+
+	private static final String IMAGE_DOCKER_IMAGE_REFERENCE = "image.dockerImageReference";
+	private static final String STATUS = "status";
+	private static final String STATUS_IMAGES = "status.images";
+	private static final String FROM_KIND = "from.kind";
+	private static final String SPEC_IMAGES = "spec.images";
+	private static final String SPEC_IMPORT = "spec.import";
+
+	public ImageStreamImport(ModelNode node, IClient client, Map<String, String[]> overrideProperties) {
+		super(node, client, overrideProperties);
+	}
+
+	@Override
+	public void setImport(boolean importTags) {
+		set(SPEC_IMPORT, importTags);
+	}
+
+	@Override
+	public boolean isImport() {
+		return asBoolean(SPEC_IMPORT);
```

Comment:
whats the meaning of this method? What does it answer? (it's missing javadoc in the interface)